model/work.py

```diff
@@ -874,10 +874,9 @@ def calculate_presentation_edition(self, policy=None):
         )
         return changed
 
-
     def calculate_presentation(
         self, policy=None, search_index_client=None, exclude_search=False,
-        default_fiction=None, default_audience=None
+        default_fiction=None, default_audience=Classifier.AUDIENCE_ADULT
```

Comment:
@leonardr, I set `default_audience` to `Classifier.AUDIENCE_ADULT` instead of implementing [SIMPLY-2610](https://jira.nypl.org/browse/SIMPLY-2610)

Reply:
I think this is OK. Since this code was written we have had many incoming feeds with no audience information and the implication was always that the works are for adults.

Reply:
Christina pushed back a little in https://jira.nypl.org/browse/SIMPLY-2610; I'm waiting for her response. It may be better to simply implement 2610.

Another temporary way to solve this problem would be to continue to set audience=None for these books but to change the way books are placed into lanes. Specifically, we could say that a lanes whose audiences include AUDIENCE_ADULT or AUDIENCE_RESEARCH should also include any books that have no audience set.

The real problem we're trying to address is not that these books don't have an audience set (that's the most accurate conclusion -- we don't have the information) but that they don't show up in the default lanes.

Reply:
Based on the conversation with Christina in https://jira.nypl.org/browse/SIMPLY-2610, I don't think users will accept a change like this which adds metadata that was not provided by the vendor, the administrator, or a librarian.

My preference would be to resolve this by resolving SIMPLY-2610.

Reply:
Let me make this stronger: based on Christina's feedback I think you need to implement SIMPLY-2610 instead of setting this default.

Reply:
Sure, will do

Reply:
I created [PR # 1214 in server_core](https://github.com/NYPL-Simplified/server_core/pull/1214) and merged it to this branch.